apis/guides/Current Member Data Stub.md

```diff
@@ -0,0 +1,3 @@
+# Current Member Data
+
+To return information about the current site visitor, use the [Get Current Member](https://dev.wix.com/docs/sdk/backend-modules/members/members/get-current-member) method. The Current Member Data API is deprecated but you still can read the [article](https://dev.wix.com/docs/client/api-reference/deprecated/apis-deprecated/members/current-member-data) about it.
```

Comment:
```suggestion
To return information about the currently logged-in site member, call [Get Current Member](https://dev.wix.com/docs/sdk/backend-modules/members/members/get-current-member). 

The legacy [Current Member Data API](https://dev.wix.com/docs/client/api-reference/deprecated/apis-deprecated/members/current-member-data) has been deprecated and will be removed on June 30, 2025.
```
I think Get Current Member works only for logged-in members, not anonymous visitors. The other edits are stylistic.